vignettes/faq.Rmd

```diff
@@ -14,6 +14,55 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup}
+```{r setup, message=FALSE, warning=FALSE}
 library(tfrmt)
+library(dplyr)
+library(gt)
+```
+
+
+### How do you add a newline within a column header?
```

Comment:
A couple of other things to mention:
1- User can also choose to use \n to get a newline
2- More generally speak to markdown formatting. Not only can we do new lines but we can have bolding and italicizing in the labels

Reply:
Done